### PR TITLE
fix(Dialog): fullscreen mask click. fix #927

### LINKED ISSUE #927: [Dialog]Fullscreen dialog cannot close by mask click
### Component 
Dialog

### Reproduction link 
[https://riddle.alibaba-inc.com/riddles/142a0581](https://riddle.alibaba-inc.com/riddles/142a0581)

### Steps to reproduce
create a dialog with isFullScreen=true and hasMask=true
click on the mask and it does not close the dialog

### PR DESCRIPTION
**Pull request by jdkahn**

### CODE REVIEW

File: src/overlay/main.scss [hunk 1]
```diff
@@ -17,12 +17,12 @@
         left: 0;
         width: 100%;
         height: 100%;
-        background: $mask-background;
+        background-color: change-color($color: $mask-background, $alpha: $mask-opacity);
```
Comment (youluna) [inline src/overlay/main.scss]: `change-color` isn't defined, is it a css method?

File: src/dialog/dialog.jsx [hunk 2]
```diff
@@ -383,19 +384,6 @@ export default class Dialog extends Component {
...
-                        dir={rtl ? 'rtl' : undefined}
-                    >
-                        {inner}
-                    </div>
-                ) : (
-                    inner
-                )}
-            </Overlay>
-        );
+        return <Overlay {...newOverlayProps}>{inner}</Overlay>;
```
Comment (youluna) [inline src/dialog/dialog.jsx]: the dom structor of full screen dialog changed, this should be a minor change, switch target branch to feat/1.17.0

File: src/overlay/main.scss (hunk 1 above)
Comment (jdkahn) [reply src/overlay/main.scss]: Its a SASS method: https://sass-lang.com/documentation/functions/color#change-color

File: src/dialog/dialog.jsx (hunk 2 above)
Comment (jdkahn) [reply src/dialog/dialog.jsx]: Is this still necessary as master is now 1.17.0?

File: src/overlay/main.scss (hunk 1 above)
Comment (youluna) [reply src/overlay/main.scss]: do you have any idea about the version of `change-color`? For some reason we have to support node-sass@4.6.0 which depends on libsass - the C++ one - 3.5.0.beta.2

**Comment by youluna** [comment]
wait for ice's adaptor https://ice.work/docs/cli/plugin-list/fusion#%E5%A4%9A%E4%B8%BB%E9%A2%98%E9%85%8D%E7%BD%AE

**Comment by youluna** [comment]
ice fixed the issue alibaba/ice#2715 yesterday, we need some adjustment refer to https://github.com/alibaba-fusion/next/issues/1029

File: src/overlay/main.scss (hunk 1 above)
Comment (youluna) [reply src/overlay/main.scss]:
```scss
// src/overlay/main.scss line20
background-color: $color-calculate-mask-background;

// src/overlay/scss/variable.scss 
// we don't have this file, you need to create one
$color-calculate-mask-background: change-color($color: $mask-background, $alpha: $mask-opacity);

```

**Pull request merged**